.claude/scripts/blocked-label-blocker-line-go/main.go

```
@@ -0,0 +1,399 @@
+					if unicode.IsControl(r) {
+						return unicode.ReplacementChar
+					}
+					return r
+				}, string(snippet))
+				fmt.Fprintf(stdout, "  >>%s", safe)
+			}
+		}
+		fmt.Fprintln(stdout)
+	}
+	if bad > 0 {
+		fmt.Fprintf(stdout, "\nblocked-label-blocker-line.sh: %d of %d open blocked-labelled issue(s) need repair (missing, malformed, or an unraised authority blocker).\n", bad, len(issues))
```

golangci-lint (2.13.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)